Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/test/TestRunner.java

```diff
@@ -0,0 +1,405 @@
+                .openSearchPage()
+                .search(searchParam)
+                .checkSearchResultsTitleAndContent(searchParam);
+
+        new MainPage()
+                .openNote(noteDTO)
+                .trash()
+                .logout(email);
+    }
+
+    @Test
+    public void testFilterByTagWhenClickingOnTagInTagDrawer() {
+        NoteDTO noteDTO = DataProvider.generateNotesWithUniqueContent(1).get(0);
+
+        new IntroPage()
+                .goToLoginWithEmail()
+                .login(email, password);
+
+        new MainPage()
+                .switchCondensedNoteListMode(false)
+                .pressBack();
+
+        new MainPage()
+                .addNewNote(noteDTO)
+                .selectTagFromDrawer(noteDTO.getTags().get(0))
+                .checkNoteContentIsInTheList(noteDTO.getContent().substring(0, 15));
+
+        new MainPage()
+                .openNote(noteDTO)
+                .trash()
+                .logout(email);
+    }
+
+    @Test
+    public void testViewTrashedNotesByClickingOnTrash() {
+        NoteDTO noteDTO = DataProvider.generateNotesWithUniqueContent(1).get(0);
+
+        new IntroPage()
+                .goToLoginWithEmail()
+                .login(email, password);
+
+        new MainPage()
+                .addNewNote(noteDTO)
+                .openNote(noteDTO)
+                .trash()
+                .openTrashPage()
+                .checkNoteIsTrashed(noteDTO.getTitle());
+
+        new MainPage()
+                .logout(email);
+    }
+
+    @Test
+    public void testRestoreNoteFromTrashScreen() {
+        NoteDTO noteDTO = DataProvider.generateNotesWithUniqueContent(1).get(0);
+
+        new IntroPage()
+                .goToLoginWithEmail()
+                .login(email, password);
+
+        new MainPage()
+                .addNewNote(noteDTO)
+                .openNote(noteDTO)
+                .trash()
+                .openTrashPage()
+                .openTrashedNote(noteDTO.getTitle())
+                .restore()
+                .openMain();
+
+        new MainPage()
+                .openNote(noteDTO)
+                .trash()
+                .logout(email);
+    }
+
+    @Test
+    public void testTrashNote() {
+        NoteDTO noteDTO = DataProvider.generateNotes(1).get(0);
+
+        new IntroPage()
+                .goToLoginWithEmail()
+                .login(email, password);
+
+        new MainPage()
+                .addNewNote(noteDTO)
+                .openNote(noteDTO)
+                .trash()
+                .checkNoteTitleIsNotInTheList(noteDTO);
+
+        new MainPage()
+                .logout(email);
+    }
+
+    @Test
+    public void testShareAnalytics() {
+
+        new IntroPage()
+                .goToLoginWithEmail()
+                .login(email, password);
+
+        new MainPage()
+                .switchShareAnalytics(true)
+                .logout(email);
+    }
+
+    @Test
+    public void testCondensedModeOpened() {
```

Comment:
I'm a little confused by the names `testCondensedModeOpened` and `testCondensedModeClosed` — on first read I thought "opened" meant "enabled"/"on" and "closed" meant "disabled"/"off" but it looks like it's the opposite. Could we clarify those names?

Reply:
Sure! It will be handled.